packages/block-editor/src/components/line-height-control/index.js

```diff
@@ -70,7 +74,7 @@ export default function LineHeightControl( { value: lineHeight, onChange } ) {
 				onKeyDown={ handleOnKeyDown }
 				onChange={ handleOnChange }
 				label={ __( 'Line height' ) }
-				placeholder={ BASE_DEFAULT_VALUE }
+				placeholder={ placeholder }
```

Comment:
I like this way of doing things. ❤️ Having the default value as the placeholder works I think as I can still set numerical values to `0` for example, and the native appearance (greyed out) indicates that it's a default value.

Just noting a couple of things down, all of which are probably best for follow ups if this gets off the ground.

- We'll have to parse unit values (`px`, `vh` etc) from the default values and reset them in the controls where units change
-  I tested a few controls, and I think there will be some work to ensure we can pass default values as placeholders to the rest of our controls hooks. For example, [box-control](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/box-control/index.js#L187) will require a tweak to handle placeholders for top/right/bottom/left inputs. This is for spacing controls such as margin and padding. But anything importing `<UnitControl />` directly such as [border-width ](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/hooks/border-width.js#L107) should just work 👍

Reply:
Good point regarding the units @ramonjd. I guess we can deal with units a follow-up to avoid growing this PR.